File: app/src/main/java/zhan/rnadomgirdanimation/SimpleHolder.java
package zhan.rnadomgirdanimation;

import android.view.View;
import android.widget.ImageView;

import androidx.recyclerview.widget.RecyclerView;

public class SimpleHolder extends RecyclerView.ViewHolder {

    private ImageView mIconIv;

    public SimpleHolder(View itemView) {
        super(itemView);

        mIconIv = (ImageView) itemView.findViewById(R.id.icon_iv);
    }

    public void bind(Integer resId) {
        mIconIv.setImageResource(resId);
    }
}

File: app/src/main/java/zhan/rnadomgirdanimation/SimpleHolder.kt
package zhan.rnadomgirdanimation

import android.view.View
import android.widget.ImageView

import androidx.recyclerview.widget.RecyclerView

class SimpleHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    private val mIconIv: ImageView

    init {

        mIconIv = itemView.findViewById<View>(R.id.icon_iv) as ImageView
    }

    fun bind(resId: Int?) {
        mIconIv.setImageResource(resId!!)
    }
}
